### PR TITLE
feat: zwave-js@14.3.0 and @zwave-js/server@1.40.0

### PR DESCRIPTION
**Pull request by AlCalzone**
I'll collect the necessary changes here for when v14 gets released.

### CODE REVIEW

File: package.json
Comment (robertsLando) [inline package.json:1]: what changes did you do on this file except the dep bump?

Comment (AlCalzone) [reply package.json:1]: AFAIK nothing. Seems to be a whitespace change.

Comment (AlCalzone) [reply package.json:1]: I'll try and clean that up before the final merge.

**Comment by AlCalzone** [comment]
Bumped the dependencies, but we're still waiting for a server release.

**Pull request merged**